### PR TITLE
Add footer base

### LINKED ISSUE #22: Create footer
### Footer Section

Create a footer simple footer section to display the copyright information.

### PR DESCRIPTION
**Pull request by raphaelanjos1**
# Description

Add footer section base.

- Creates a functional footer component, since there's no need to need to import and inherit an entire instance by now.
- Uses redirect folder as  folder separation pattern.

Fixes #22 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: frontend/src/components/organisms/footer/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from "react";
+
+const Footer = () => (
```
Comment (clucasalcantara) [inline frontend/src/components/organisms/footer/index.js]: Quando ao CSS, pode criar um style.css e importar no index do footer, no stress 😎

File: frontend/src/pages/home/index.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import React, { Component, Fragment } from "react";
 import { Link } from "react-router-dom";
+import { Footer } from "../../components";
```
Comment (clucasalcantara) [inline frontend/src/pages/home/index.js]: <3

**Comment by clucasalcantara** [comment]
Se quiser ja matar esse PR, muda o titulo pra add footer base for now! tratamos o CSS e estilo em outro PR!

**Comment by raphaelanjos1** [comment]
bele, vou mudar aqui ^^

**Pull request merged**